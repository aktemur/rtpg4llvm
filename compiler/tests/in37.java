/*compunit*/
int foo() { return 5; }
