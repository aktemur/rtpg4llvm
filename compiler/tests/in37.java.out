new FuncDeclaration(new IntType(),
      "foo",
      mkFormals(0),
      mkScopedStmt(1, new ReturnStmt(mkConst(5))))
